- 99.1.0 was to trigger auto-download of libcyaml head with some unreleased bugfix

--- meson.build.orig	2021-04-14 08:19:57 UTC
+++ meson.build
@@ -769,7 +769,7 @@ endif
 # force patched version of libcyaml from subproject
 # see https://github.com/tlsa/libcyaml/issues/150
 cyaml_dep = dependency(
-  'libcyaml', required: false, version: '>=99.1.0',
+  'libcyaml', required: false, version: '>=1.1.0',
   static: all_static)
 if not cyaml_dep.found()
   cyaml_proj = subproject('libcyaml')
